package.json

```diff
@@ -6,9 +6,9 @@
   "author": "Andrew Kirilenko, Yury Yantsevich & Iron.io, Inc",
   "main": "./lib/client",
   "dependencies": {
-    "pkginfo": "0.2.3",
-    "underscore": "1.3.3",
-    "iron_core": ">=0.2.3"
+    "pkginfo": "0.3.0",
+    "underscore": "1.4.4",
```

Comment:
what about moving it to 1.7.0, too? It means, that we must release both `iron_core` and `iron_mq` (v2 version). Alternatively, just create new issue on GH about update dependencies before v3 library release.